chaco/plots/polar_line_renderer.py

```diff
@@ -133,6 +133,7 @@ def _downsample(self):
     def _draw_plot(self, *args, **kw):
         """Draws the 'plot' layer."""
         self._gather_points()
+        gc = args[0]
```

Comment:
If you want the `gc` to be the first argument to the function, make it the first argument to the function!

Doing this is fragile, not self-documenting and will produce confusing error messages if used incorrectly.